app/forms/steps/client/contact_details_form.rb

```diff
@@ -8,14 +8,26 @@ class ContactDetailsForm < Steps::BaseFormObject
 
       attribute :telephone_number, :string
 
+      attribute :correspondence_address_type, :correspondence_address_type
```

Comment:
As mentioned, we can use just a `:string` and not create a custom type just for this. There is no real benefit as we are not doing transformations or anything like that 👍